package.json

```diff
@@ -24,7 +24,7 @@
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
     "compile": "tsc -p .",
     "format-check": "./bin/format-check.sh",
```

Comment:
Can this be just `prettier --list-different glob` now?

Reply:
Yes it can!  Good call out 👍